managing_providers/_topics/embedded_workflows.md

```diff
@@ -174,12 +174,89 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
+      * `VerifyHostname` - Boolean - Verify SSL certificate hostname.
+      * `Hostname` - String - Server hostname for SNI.
+      * `CaFile` - String - Path to a CA file in PEM format.
+      * `CaPath` - String - Path to a CA directory.
+      * `VerifyMode` - Integer - OpenSSL constant.
+      * `VerifyDepth` - Integer - Maximum depth for the certificate chain validation.
+      * `Version` - Integer - SSL Version.
+      * `MinVersion` - Integer - Minimum SSL Version.
+      * `MaxVersion` - Integer - Maximum SSL Version.
+      * `Ciphers` - String - Ciphers supported.
+    * `Options`
+      * `Timeout`
+      * `ReadTimeout`
+      * `OpenTimeout`
+      * `WriteTimeout`
+      * `Encoding` - String
+        * `JSON` - JSON encodes the request and decodes the response
+    * `Proxy`
+      * `Uri` - String - URI of the proxy.
+      * `User` - String - User for the proxy.
+      * `Password` - String - Pasword for the proxy
+
+
+  * `manageiq://email` - Send an email using the configured SMTP server
+
+    Parameters:
+    * `To` - Array of recipient email addresses, defaults to service requester email
+    * `From` - Sender email address, defaults to smtp.from Setting
+    * `Subject` - Email Subject string
+    * `Cc` - Array of recipients to carbon-copy
+    * `Bcc` - Array of recipients to blind-carbon-copy
+    * `Body` - The body of the email
+    * `Attachment` - A hash with the filename as the key and the content as the value
+
+  * manageiq://embedded_ansible - Execute an ansible playbook with EmbeddedAnsible
+
+    Identifying a playbook: You must identity a playbook by either:
+     `PlaybookId` - This is the database identifier of the `ConfigurationScript`
+
+    or
+
+    `RepositoryUrl`, `RepositoryBranch`, and `PlaybookName`
+
+    Parameters:
+    * `RepositoryUrl` - URL of the configuration script source identifying the repository where the playbook resides
+    * `RepositoryBranch` - Branch of the configuration script source where the playbook resides
+    * `PlaybookName` - Name of the playbook
+    * `PlaybookId` - ID of the `ConfigurationScript`
+    * `Hosts` - Array of hostnames to target with the playbook
+    * `ExtraVars` - Hash with key/value pairs that will be passed as extra_vars
+    * `BecomeEnabled` - Boolean if playbook should activate privilege escalation, defaults to false
+    * `Timeout` - Integer in minutes for how long to allow the playbook to run for
+    * `Verbosity` - Integer ansible verbosity level 0-5
+    * `CredentialId` - ID of an ansible credential
+    * `CloudCredentialId` - ID of an ansible cloud credential
+    * `NetworkCredentialId` - ID of an ansible network credential
+    * `VaultCredentialId` - ID of an ansible vault credential
+
+  * manageiq://provision_execute - Execute an MiqProvision task
+
+    This can be used for a VM Provision Service Catalog item in place of automate.  No parameters are required, state input is used as the provision options.
```

Comment:
```suggestion
    This can be used for a VM Provision Service Catalog item in place of automate.  No explicit parameters are required, as state input is used as the provision options.
```